common/web/keyboard-processor/src/keyboards/activeLayout.ts

```diff
@@ -778,7 +768,11 @@ export class ActiveLayer implements LayoutLayer {
 }
 
 export class ActiveLayout implements LayoutFormFactor{
-  layer: ActiveLayer[];
+  /**
+   * Holds all layer specifications for the layout.  There is no guarantee that they
+   * have been fully preprocessed.
+   */
+  layer: TouchLayerSpec[];
```

Comment:
Rename?

```suggestion
  layers: TouchLayerSpec[];
```

Reply:
So... this is a request to refactor the variable to be plural?  There's no bug at the moment - the variable is currently `layer`, not `layers`.

Reply:
If we were to do this, it'd likely be best as a separate PR, just to keep things extra clear.